### ISSUE TITLE
Webhook not triggered on bulkDelete

### ISSUE THREAD

**Issue by Keith-Hon** [strapi/strapi#12290]
### Steps to reproduce the behavior

1. Go to Content Manager in admin panel
2. Click on the select-all box next to "ID"
3. Click "Delete" to bulk delete all selected items
4. Webhook is not triggered

### Expected behavior
A webhook should be triggered with the entries deleted

### System

- Node.js version: v12.22.8
- NPM version: 6.14.15
- Strapi version: v4.0.3
- Database: mysqlite
- Operating system: ubuntu 20.04.2 LTS

**Comment by derrickmehaffy** [comment]
Confirmed on v4.0.5

**Comment by avtomatron** [comment]
Hi! Still no updates about this issue?

**Comment by vieweg** [comment]
Same here, any idea if is someone working on it?

**Comment by derrickmehaffy** [comment]
No, this is not currently scheduled in any squad. Please feel free to open a PR.
